app/assets/stylesheets/_responsive.css.scss

```diff
@@ -6,8 +6,34 @@
   nav .inner-left {
     @include span-columns(3 of 5);
 
+    .mobile-control {
+      cursor: pointer;
+      display: inline-block;
+      padding-right: 10px;
+    }
+
     .control {
-      display: none;
+      background-color: #3c3c3c;
```

Comment:
Color literals like `#3c3c3c` should only be used in variable declarations; they should be referred to via variable everywhere else.